WORKSPACE

```diff
@@ -47,7 +47,7 @@ go_rules_dependencies()
 
 go_download_sdk(
     name = "go_sdk",
-    version = "1.20.10",
+    version = "1.20.12",
 )
 
 go_register_toolchains()
```

Comment:
Down in lines 78-85, we appear to be pulling the `go-runner` at `v2.3.1-go1.20.4-bullseye.0`.  This wasn't updated by previous Go bumps, but being relatively unfamiliar with this repo, I'm not sure if this was deliberate or an oversight.  Please advise as to whether that should also be updated.

Reply:
IMO, no harm in updating it, but not all deployments use go-runner

Reply:
We should update it.

Reply:
Done.  `--forced` push to keep history minimal.